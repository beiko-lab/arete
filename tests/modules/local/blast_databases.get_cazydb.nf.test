nextflow_process {

    name "Test Process GET_CAZYDB"
    script "modules/local/blast_databases.nf"
    process "GET_CAZYDB"

    test("Can download CAZYDB") {

        then {
            assert process.success
            assert process.trace.tasks().size() == 1
            assert snapshot(process.out).match()
            with(process.out) {
              // check if emitted output has been created
              assert cazydb.size() == 1
            }
        }

    }

}
