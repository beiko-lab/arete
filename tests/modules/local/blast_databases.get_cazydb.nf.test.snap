{
    "Can download CAZYDB": {
        "content": [
            {
                "cazydb": [
                    "CAZyDB.07312020.fa:md5,5a358a7761aaea37a61b59f5deba18d9"
                ]
            }
        ],
        "timestamp": "2023-02-06T12:01:39+0000"
    }
}